#! /bin/sh /usr/share/dpatch/dpatch-run
## 00_installscript.dpatch by Ignace Mouzannar <mouzannar@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: clean the install_script.sh script provided by upstream

@DPATCH@
diff -urNad aqemu-0.7.2~/install_script.sh aqemu-0.7.2/install_script.sh
--- aqemu-0.7.2~/install_script.sh	2009-06-23 21:09:27.000000000 +0200
+++ aqemu-0.7.2/install_script.sh	2009-06-23 21:10:42.000000000 +0200
@@ -5,60 +5,8 @@
 # Author: Denis Smirnov (aka reMiND)
 # Author: Michael Brandstetter
 
-# Print Help
-if [ "$1" = "--help" ]; then
-	echo "For change install directory run: ./install_script.sh path_to_you_dir"
-	exit 0
-fi
-
-# User Set Other Installation Dir?
-if [ -n "$1" ]
-then
-	DESTDIR=$1
-	echo "Install data path changed to: $1"
-fi
-
-# Default Installation Dirs For Linux
-PREFIX=/usr
-
-# Set Installation Dirs For FreeBSD
-if uname -a | grep "FreeBSD" > /dev/null;
-then
-	PREFIX=/usr/local
-fi
-
-BINDIR=$PREFIX/$DESTDIR/bin
-DATADIR=$PREFIX/$DESTDIR/share
-
-# Unistall Previous AQEMU Version
-echo "Find and remove a previous AQEMU installation..."
-
-FIND_PREV="no"
-
-if [ -x /usr/share/aqemu ]
-then
-	FIND_PREV="yes"
-	echo "Remove /usr/share/aqemu directory..."
-	rm -rf /usr/share/aqemu
-fi
-
-if [ -x /usr/share/apps/aqemu ]
-then
-	FIND_PREV="yes"
-	echo "Remove /usr/share/apps/aqemu directory..."
-	rm -rf /usr/share/apps/aqemu
-fi
-
-if [ -x /usr/local/share/aqemu ]
-then
-	FIND_PREV="yes"
-	echo "Remove /usr/local/share/aqemu directory..."
-	rm -rf /usr/local/share/aqemu
-fi
-
-if test "$FIND_PREV" = "no"; then
-	echo "No previous AQEMU versions found."
-fi
+BINDIR=$DESTDIR/usr/bin
+DATADIR=$DESTDIR/usr/share
 
 # Copy New Data Files
 echo "Installing new AQEMU files..."
@@ -78,14 +26,6 @@
 	bzip2 -9 -c $doc_file > "$DATADIR/doc/aqemu/$doc_file.bz2"
 done
 
-echo "Create menu items..."
-install -d $DATADIR/pixmaps/
-install -Dm 644 ./menu_data/*.png $DATADIR/pixmaps/
-install -Dm 644 ./menu_data/aqemu.desktop $DATADIR/applications/aqemu.desktop
-
-install -Dm 644 ./menu_data/aqemu $DATADIR/menu/aqemu
-echo "  command=\"$BINDIR/aqemu\" \\" >> $DATADIR/menu/aqemu
-echo "  icon=\"$DATADIR/pixmaps/aqemu_48x48.png\" \\" >> $DATADIR/menu/aqemu
-
 echo "Installation complete!"
+
 exit 0
